Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -981,6 +981,9 @@ public class PBXProjGenerator {
                                 queue.append(dependencyTarget)
                             }
                         }
+                    } else {
```

Comment:
Would be good if the code showed what was happening as well as the comment. For completion sake could we add a `Project.getAggregateTarget(_ name: String) -> AggregateTarget` function and use it here
```swift
} else if project.getAggregateTarget(dependency.reference) != nil {
```